src/encoded/static/components/search.js

```diff
@@ -1050,10 +1041,9 @@ export class FacetList extends React.Component {
         // are the negation facet terms that need to get merged into the regular facets that their
         // non-negated versions inhabit.
         const negationFilters = filters.filter(filter => filter.field.charAt(filter.field.length - 1) === '!');
-        
-        console.log("building facets");
-        
-        console.log(width);
+
+        // If there are no assay terms, the annotation type facet will need the "Method" label
+        const assayTerms = facets.filter(facet => facet.field === 'assay_term_name').some(facet => facet.terms.some(f => f.doc_count > 0));
```

Comment:
Could this use `.find()` instead of `.filter()`? Or can there be more than one assay_term_name facet?

If you can use `.find()`, you’ll probably need to break this into two or three lines in case `.find()` doesn’t find anything, but that might be good because I’m having a tough time wrapping my head around this line. I _think_ that way you would only need one `.some()`.